Task_DEV_5/Task_DEV_5/SecondSorter.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Text;
+using System.IO;
+
+namespace task_DEV_5
+{
+    class SecondTextSort
+    {
+        private string path { get; set; }
+        public FileStream fstream;
+
+        public SecondTextSort(string path)
+        {
+            this.path = path;
+        }
+
+        public string ReadLine()
```

Comment:
This method should be private. Because if you try to call this method outside there will be error because your filestram will be closed.

Reply:
Done.